pymatgen/io/qchem/outputs.py

```diff
@@ -154,6 +157,9 @@ def __init__(self, filename: str):
             self.data["solvent_data"]["PCM_dielectric"] = float(temp_dielectric[0][0])
             self._read_pcm_information()
         elif self.data["solvent_method"] == "SMD":
+            # TODO - is this a bug? It appears that the code block below if the string
```

Comment:
The first "if" only proceeds if "Unrecognized solvent" is found. The second "if" only proceeds if the calculation failed to complete. We're differentiating between setting "unrecognized_solvent" as an error if the calc failed to complete or a warning if it was able to complete. I believe this is correct as-is.

Reply:
But the first `if` tests whether `read_pattern` returns `[[]]`. Won't that only be the case if `Unrecognize Solvent` is NOT found in the output?

Reply:
OK, if the match is not found it would return `[]` not `[[]]`. Makes sense now.